src/server/req_delete.c

```diff
@@ -421,6 +421,8 @@ req_deletejob(struct batch_request *preq)
 		}
 
 		if ((i == JOB_STATE_EXITING) && (forcedel == 0)) {
+			if (parent->ji_pmt_preq != NULL) 
+				reply_preempt_jobs_request(PBSE_BADSTATE, PREEMPT_METHOD_DELETE, parent);
```

Comment:
I didn't really need to handle this case since you'll never get here.  If the job is not in the running state, you don't do a delete job request.  You handle the preemption right away.  I did as you suggested since I could though.  I did a find_job()